CONTRIBUTING.rst

```diff
@@ -11,6 +11,13 @@ Pull requests are great! We love them; here is a quick guide:
   contribute but don't know how to begin, the `good-first-bug`_ label 
   of the `issue tracker`_ is the place to go. 
   (If you're new to Git: `Start Here`_)
+  
+- Before contributing make sure you checkout the docs. There are two versions of docs available 
+
+  + `latest`_, for use with bleeding-edge github version.
```

Comment:
Ryan, while I'm thankful that you're looking out for typos, I think that when it comes to minor changes like writing issues, it's better to provide a patch than to write comments. (You can also push directly to the author's branch if the author has enabled that, which is the default, I think.)